src/Gradus.jl

```diff
@@ -354,6 +354,19 @@ Base.length(::AbstractCoronaModel) = 1
 Base.iterate(m::AbstractCoronaModel) = (m, nothing)
 Base.iterate(::AbstractCoronaModel, ::Nothing) = nothing
 
+"""
+TODO: DOCSTRING
+"""
+abstract type AbstractCoronalSpectrum end
+
+# for broadcasting
+Base.length(::AbstractCoronalSpectrum) = 1
+Base.iterate(g::AbstractCoronalSpectrum) = (g, nothing)
+Base.iterate(::AbstractCoronalSpectrum, ::Nothing) = nothing
```

Comment:
This is just so that we can do things like `f.(spectrum, array)`. If Julia doesn't know the length of something, it wont let you broadcast.